RobotsAndPencils/react-gantry
# Update templates

Fixes #30

- Updates the Container Generator to import duck actions automatically correctly
- Updates Component generator to match the new container's class version
- Updates Duck generator to accomodate the removed index.js file
- Updates newAction generator to accomodate removed index.js file and shifted export scheme
- Adds Thunk Generator within newAction (add `--thunk` and possibly `--promise` if you want a promise thunk)

To Test:
- Basically the best way to test this is to try all of the iterations of flags within each modified generator and see if:
  - anything breaks horribly
  - the generated files and actions look correct
